core/src/lib.rs

```diff
@@ -115,8 +111,8 @@ impl SP1Prover {
     }
 }
 
-impl SP1Verifier {
-    /// Verify a proof generated by `SP1Prover`.
+impl VerifierClient {
+    /// Verify a proof generated by `ProverClient`.
```

Comment:
IMO, it's more clear to combine the methods from the `ProverClient` and `VerifierClient` into an `SP1Client`, as there's no hosted/local delineation in the `VerifierClient`.

Reply:
And also simplifies imports.

Reply:
[merged VeriferClient into ProverClient](https://github.com/succinctlabs/sp1/pull/483/commits/a1c37d74a46d0f569fe85c909dafff5f7d4211ef)